### DISCUSSION TITLE
Survey - Front-end 2018

### DISCUSSION THREAD

**Discussion by felipefialho** [frontendbr/forum#926] [Discussão]
Fala amigs, como estão? :)

Em [2014](https://www.felipefialho.com/survey/2014/ ) e [2016](https://www.felipefialho.com/survey/ ) rodei um survey p/ gente ter uma noção salarial de frontenders no Brasil.

A ideia esse ano é ser mais abrangente, para isso preciso de MUITAS respostas.

Bora responder e ajudar na divulgação?

https://lfeh.typeform.com/to/Uh07u5

**Comment by rubenmarcus** [comment]
De um ano pra cá eu senti que a média salarial aumentou bastante... mesmo que as vagas CLT,estejam cada vez mais raras, e o número de consultorias explodiu..
Algumas vagas pagam um salário até alto, (considerando 2 ou 3 anos atrás), mas sem beneficios nenhum, e seco, com empresa limitada com 15% de desconto (isso somado aos encargos, se você for pensar em pagar transporte, comida, plano de saúde etc , vai quase 30 - 40% do salário)

A demanda aumentou MUITO mesmo, assim como o número de profissionais e a facilidade de material que temos hoje.

Eu espero que isso continue assim pelo menos uns 10 anos aí, porquê to vendo que tem muito código legado e cada vez mais acho que vai ter, a um ano atrás eu tava meio em cima do muro pensando se devia virar fotografo, ou tatuador ou ficar como front mesmo, e decidi por front, e também de não ter mais preguiça de estudar JS, porque vi as possibilidades que o JS traz hoje em dia, como por exemplo, **_a menina da ThoughtWorks, controlando um cubo no browser com a mente_**... 


Espero que o mercado brasileiro amadureça, porquê temos uma das comunidades mais ativas no mundo ,e os devs mais talentosos, infelizmente a média salarial aqui é muito ruim ainda, e a estrutura também, minha visão é que a Europa, Eua e Ásia dão um banho em investimento, e capital humano no nosso mercado, e continuamos sendo apenas exportador de cerébros ( conheço muitos amigos que foram embora daqui nos últimos 3 anos)

Fui pra Índia 2 vezes desde 2014, e vi que o mercado lá é 10x maior que o nosso, e por engraçado que seja, quem é junior ou estagiário lá ganha um salário de miséria, mas quem se torna pleno ou senior ganha até 2x o que ganhamos aqui, e tem muita vaga pra Designer, UI Developers e Front muito mais, especialmente nas empresas grandes lá ou startups, o governo também está investindo forte em bolsões de tecnologia, e a maioria dos profissionais não fica lá, e vai pra Alemanha,Inglaterra ou EUA. Eu todo dia tinha oportunidades de entrevista lá, não fui porque estava com a cabeça pra passear, e também porque não troco a comida brasileira por nada <3

**Comment by rubenmarcus** [comment]
Sobre a média salarial minha percepção veio do Linkedin. Quando antes eu pedia pretensão mais que 5k, muitas vagas nem respondiam. Hoje em dia, a maioria acha normal. E pretensão entre 5 e ~12-15, muitas respondem.. a uns 2 anos atrás, com requisitos parecidos de hoje, eram bem mais raro de se pagar mais que 5k..  Eu via pelo menos com as pessoas que eu conhecia, que isso era uma realidade mais pra Backend, ou Fullstack, galera de Java, C# e etc.. Hoje conheço muito front que ganha mais que back...

Hoje ainda vejo também muita vaga querendo pagar 3 dizendo que é pra junior ou pleno, pedindo skills de senior.. Mas não são a maioria.. pelo menos no que vi nos sites de vaga em geral

**Comment by lucasvalhos** [comment]
Respondido e parabéns pela iniciativa.

**Comment by kivervinicius** [comment]
Respondido, excelente iniciativa.
